### PR TITLE
RFBA test

### PR DESCRIPTION
**Pull request by marouenbg**
*(Note: You may replace [ ] with [X] to check the box)*

**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [contributing](https://github.com/opencobra/cobratoolbox/blob/documentation/.github/CONTRIBUTING.md) document
- [x] Followed the [code styleguide](https://github.com/opencobra/cobratoolbox/blob/documentation/.github/CONTRIBUTING.md#code-styleguide)
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/opencobra/cobratoolbox/pulls) for the same update/change
- [x] Written platform-independent code
- [x] Used `pwd` to get the current directory
- [x] Used `filesep` for paths (e.g., `['myPath' filesep 'myFile.m']`)
- [x] Made sure that computational results are reproducible

#### Previous output

[*Include here information of the current version of The COBRA Toolbox*]

#### Output of PR-merged version

[*Include here information of modified version of The COBRA Toolbox after your proposed PR is accepted*]

###  Documentation updates

[*Include here information on how the documentation has been updated*]

**I hereby confirm that I have:**

- [ ] Documented your code based on the [Documentation styleguide](#documentation-and-comments-styleguide).
- [ ] Documented the `Input` and `Output` arguments
- [ ] Followed the guidelines to automatically generate the documentation

###  Tests tailored to test the PR

[*Include here information on how new tests and/or how existing tests have been adapted*]

**I hereby confirm that I have:**

- [x] Written a sensible test according to the [Test styleguide](#test-styleguide)
- [x] Tested your PR locally prior to submission
- [x] Checked that all tests pass
- [x] Tested the code on Linux

### CODE REVIEW

**Comment by syarra** [comment]
@marouenbg, What is going on here?

File: test/verifiedTests/testrFBA/testOptimizeRegModel.m [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+%Solver packages
+solverPkgs = {'tomlab_cplex'};
+%QP solvers
+QPsolverPkgs = {'tomlab_cplex'};
+ 
+for k =1:length(solverPkgs)
+    
+    for j=1:length(QPsolverPkgs)%QP solvers
+        
+        % add the solver paths (temporary addition for CI)
```
Comment (laurentheirendt) [inline test/verifiedTests/testrFBA/testOptimizeRegModel.m]: The path of the `tomlab_cplex` solver should be included here.

File: test/verifiedTests/testrFBA/testOptimizeRegModel.m [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            assert(isequal(DRgenestest,DRgenes))
+            assert(isequal(constrainedRxnstest,constrainedRxns))
+            assert(isequal(statestest,states))
+            %only compare objective because solution may vary
+            assert( FBAsolstest{1}.f - FBAsols{1}.f < tol)
+        end
+    end
+    
+        % remove the solver paths (temporary addition for CI)
+%         if strcmp(solverPkgs{k}, 'tomlab_cplex')
```
Comment (laurentheirendt) [inline test/verifiedTests/testrFBA/testOptimizeRegModel.m]: The path of the `tomlab_cplex` solver should be removed here.

File: test/verifiedTests/testrFBA/testSolveBooleanRegModel.m [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ 
+%Solver packages
+solverPkgs = {'tomlab_cplex','glpk'};
+
+load modelReg;
+load testDataSolveBooleanRegModel;
+ 
+
+ for k =1:length(solverPkgs)
+    solverLPOK = changeCobraSolver(solverPkgs{k});
```
Comment (laurentheirendt) [inline test/verifiedTests/testrFBA/testSolveBooleanRegModel.m]: The solver path of `tomlab_cplex` must be included here.

File: test/verifiedTests/testrFBA/testdynamicRFBA.m [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+%Solver packages
+solverPkgs = {'tomlab_cplex'};
+%QP solvers
+QPsolverPkgs = {'tomlab_cplex'};
+
+for k =1:length(solverPkgs)
+    solverLPOK = changeCobraSolver(solverPkgs{k},'LP');
+    for j=1:length(QPsolverPkgs)%QP solvers
+        solverQPOK = changeCobraSolver(QPsolverPkgs{j},'QP');
```
Comment (laurentheirendt) [inline test/verifiedTests/testrFBA/testdynamicRFBA.m]: The solver path of `tomlab_cplex` must be included here.

File: test/verifiedTests/testrFBA/testOptimizeRegModel.m [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+% Purpose:
+%     - testrFBA tests the optimizeRegModel function and its different outputs
+%
+% Author:
+%     - Marouen BEN GUEBILA - 31/01/2017
+ 
+ 
+% define global paths
+global path_TOMLAB
+global path_GUROBI
```
Comment (laurentheirendt) [inline test/verifiedTests/testrFBA/testOptimizeRegModel.m]: `path_GUROBI` not necessary if not used.

File: test/verifiedTests/testrFBA/testSolveBooleanRegModel.m [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+%
+% Author:
+%     - Marouen BEN GUEBILA - 31/01/2017
+ 
+ 
+% define the path to The COBRAToolbox
+pth = which('initCobraToolbox.m');
+CBTDIR = pth(1:end - (length('initCobraToolbox.m') + 1));
+ 
+cd([CBTDIR '/test/verifiedTests/testrFBA'])
```
Comment (laurentheirendt) [inline test/verifiedTests/testrFBA/testSolveBooleanRegModel.m]: Did you try using `initTest()`?

File: test/verifiedTests/testrFBA/testdynamicRFBA.m [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+%
+% Author:
+%     - Marouen BEN GUEBILA - 31/01/2017
+ 
+ 
+% define the path to The COBRAToolbox
+pth = which('initCobraToolbox.m');
+CBTDIR = pth(1:end - (length('initCobraToolbox.m') + 1));
+ 
+cd([CBTDIR filesep 'test' filesep 'verifiedTests' filesep 'testrFBA'])
```
Comment (laurentheirendt) [inline test/verifiedTests/testrFBA/testdynamicRFBA.m]: Did you try using `initTest()`?

File: test/verifiedTests/testrFBA/testdynamicRFBA.m (hunk 4 above)
Comment (marouenbg) [inline test/verifiedTests/testrFBA/testdynamicRFBA.m]: I left the lines commented for you to add them with initTest and all the tricks. 
otw the test passes.

**Pull request merged**